dev/com.ibm.ws.jsonb_fat/test-applications/jsonbapp/src/web/jsonbtest/JSONBTestServlet.java

```diff
@@ -196,7 +196,8 @@ public void testArrays() throws Exception {
             byte[] b = jsonb.fromJson(cr, byte[].class);
             assertArrayEquals(bytes, b);
 
-            cr.reset();
+            //Yasson 3.0.3 now makes sure to close parser and the stream associated with a CharArrayReader
+            cr = new CharArrayReader(cw.toCharArray());
```

Comment:
```suggestion
            cr.close();
            cr = new CharArrayReader(cw.toCharArray());
```
When running with jsonb 1.0 and 2.0 do you want to have a cr.close()?

Reply:
Yes this would be the safe thing to do.  Updated.